core/src/main/java/org/opensearch/sql/analysis/Analyzer.java

```diff
@@ -174,7 +174,7 @@ public LogicalPlan visitRelation(Relation node, AnalysisContext context) {
 
   @Override
   public LogicalPlan visitRelationSubquery(RelationSubquery node, AnalysisContext context) {
-    LogicalPlan subquery = analyze(node.getChild().get(0), context);
+    LogicalPlan subquery = analyze(node.getChild().getFirst(), context);
```

Comment:
please create a method `visitFirstChild(node)` and replace all the `node.getChild().getFirst()` calls